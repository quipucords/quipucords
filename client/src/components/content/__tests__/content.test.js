import React from 'react';
import { shallow } from 'enzyme';
import Content from '../content';

describe('ConfirmationModal Component', function() {
  it('should shallow render a basic component', () => {
    const wrapper = shallow(<Content />);

    expect(wrapper).toMatchSnapshot();
  });
});
