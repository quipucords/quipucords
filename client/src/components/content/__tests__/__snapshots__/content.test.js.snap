// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfirmationModal Component should shallow render a basic component 1`] = `
<div
  className="quipucords-content"
>
  <Switch>
    <Route
      component={[Function]}
      key="/sources"
      path="/sources"
    />
    <Route
      component={[Function]}
      key="/scans"
      path="/scans"
    />
    <Route
      component={[Function]}
      key="/credentials"
      path="/credentials"
    />
    <Redirect
      from="/"
      push={false}
      to="/sources"
    />
  </Switch>
</div>
`;
